src/py42/services/users.py

```diff
@@ -367,6 +367,6 @@ def update_user(
             u"firstName": first_name,
             u"lastName": last_name,
             u"notes": notes,
-            u"quota": quota,
+            u"quotaInBytes": archive_size_quota_bytes,
```

Comment:
@ceciliastevens I believe this is supposed to be `quotaInBytes`, correct?  `quota` was not updating anything whereas this property does.